reference/5.1/Microsoft.PowerShell.Core/About/about_Continue.md

```diff
@@ -1,28 +1,31 @@
 ---
 keywords: powershell,cmdlet
-ms.date: 5/9/2019
+ms.date: 06/04/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_continue?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: about_Continue
 ---
 # About Continue
 
-## SHORT DESCRIPTION
+## Short description
 
-Describes how the `Continue` statement immediately returns the program flow
-to the top of a program loop.
+Describes how the `continue` statement immediately returns the program flow
+to the top of a program loop, a switch statement, or a trap statement.
 
-## LONG DESCRIPTION
+## Long description
 
-In a script, the `Continue` statement immediately returns the program flow
-to the top of the innermost loop that is controlled by a `For`, `Foreach`,
-or `While` statement.
+The `continue` statement provides a way to exit the current control block but
+continue execution, rather than exit completely. The statement supports labels.
+A label is a name you assign to a statement in a script.
 
-The `Continue` keyword supports labels. A label is a name you assign to a
-statement in a script. For information about labels, see
-[about_Break](about_Break.md).
+## Using continue in loops
 
-In the following example, program flow returns to the top of the While loop
+An unlabeled `continue` statement immediately returns the program flow to
+the top of the innermost loop that is controlled by a `For`, `Foreach`, `Do`,
+or `While` statement. The current iteration of the loop is terminated and the
+loop continues with the next iteration.
+
+In the following example, program flow returns to the top of the `While` loop
```

Comment:
Shouldn't these codefenced elements be lower case to match about_Break?

Reply:
Fixed